src/main.rs

```diff
@@ -412,7 +416,7 @@ fn show_impl(repo: &Repository, matches: &clap::ArgMatches) -> i32 {
         if matches.is_present("initial") {
             vec![(TreeGraphElemLine::empty(), try_or_1!(repo.find_commit(issue)))]
         } else {
-            try_or_1!(repo.get_issue_revwalk(issue))
+            try_or_1!(repo.find_issue(issue).and_then(|issue| issue.message_revwalk()))
```

Comment:
You can put the `and_then()` call on a new line here.

Also (I'm not sure with this one) you could use `and_then(Issue::message_revwalk)` which would result in a bit cleaner code.